### PR TITLE
test: use mocha

### LINKED ISSUE #26: Introduce mocha

### PR DESCRIPTION
**Pull request by syuhei176**
still wip

Close #26

### CODE REVIEW

**Comment by syuhei176** [comment]
TODO

* [ ] write all tests
* [ ] parse content
* [ ] remove simple tester js

File: test/helper/invokeTest.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const childProcess = require('child_process')
+
+function getCommand({
+  parameter,
+  initialStorage,
+  contractPath = 'contracts/main.ligo',
+  entryPoint = 'main',
+  source = 'tz1TGu6TN5GSez2ndXXeDX6LgUDvLzPLqgYV'
+}) {
+  const command = `ligo dry-run ${contractPath} ${entryPoint} '${parameter}' '${initialStorage}' --amount=1 --source=${source} --format=json`
```
Comment (shogochiai) [inline test/helper/invokeTest.js]: `--format-json` 🔥🔥🔥

File: test/deposit.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+)`)
+
+    it('deposit 1 tz', () => {
+      const result = invokeTest({
+        parameter: testParams,
+        initialStorage
+      })
+      assert.equal(result.status, 'ok')
+      // TODO: parse content
+      // assert.equal(result.content, initialStorage)
```
Comment (shogochiai) [inline test/deposit.test.js]: If we can do this parsing, even the continuous scenario testing is possible 🙏✨

File: test/commitment.test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const assert = require('assert')
+const invokeTest = require('./helper/invokeTest')
+const { rmWhiteSpaces, initialStorage } = require('./helper/utils')
+
+describe('CommitmentContract', function() {
+  this.timeout(5000)
+
+  describe('Deposit', () => {
```
Comment (daiki44) [inline test/commitment.test.js]: Commitment

**Pull request merged**